app/src/main/res/raw/data.json

```diff
@@ -11170,18 +11093,1151 @@
+        }
+    ],
+    [{
+        "EN": {
+            "word": "wear",
+            "topics": ["dress"],
+            "level": "beginner(A1)"
+        }
+    },
+        {
+            "RU": {
+                "word": "носить",
+                "topics": ["одежда и обувь"]
+            }
+        }
+    ],
+		[{
+				"EN": {
+					"word": "coat",
+					"topics": ["dress"],
+					"level": "beginner(A1)"
+				}
+			},
+			{
+				"RU": {
+					"word": "пальто",
+					"topics": ["одежда и обувь"]
+				}
+			}
+		],
+		[{
+				"EN": {
+					"word": "dress",
+					"topics": ["dress"],
+					"level": "beginner(A1)"
+				}
+			},
+			{
+				"RU": {
+					"word": "платье",
+					"topics": ["одежда и обувь"]
+				}
+			}
+		],
+    [{
+        "EN": {
+            "word": "dress",
+            "topics": ["dress"],
+            "level": "beginner(A1)"
+        }
+    },
+        {
+            "RU": {
+                "word": "одеваться",
+                "topics": ["одежда и обувь"]
+            }
+        }
+    ],
+        [{
+				"EN": {
+					"word": "belt",
+					"topics": ["dress"],
+					"level": "elementary(A2)"
+				}
+			},
+			{
+				"RU": {
+					"word": "пояс",
+					"topics": ["одежда и обувь"]
+				}
+			}
+		],		[{
+				"EN": {
+					"word": "button",
+					"topics": ["dress"],
+					"level": "elementary(A2)"
+				}
+			},
+			{
+				"RU": {
+					"word": "пуговица",
+					"topics": ["одежда и обувь"]
+				}
+			}
+		],		[{
+				"EN": {
+					"word": "comfortable",
+					"topics": ["dress"],
+					"level": "elementary(A2)"
+				}
+			},
+			{
+				"RU": {
+					"word": "удобный",
+					"topics": ["одежда и обувь"]
+				}
+			}
+		],		[{
+				"EN": {
+					"word": "designer",
+					"topics": ["dress"],
+					"level": "elementary(A2)"
+				}
+			},
+			{
+				"RU": {
+					"word": "дизайнер",
+					"topics": ["одежда и обувь"]
+				}
+			}
+		],		[{
+				"EN": {
+					"word": "fashion",
+					"topics": ["dress"],
+					"level": "elementary(A2)"
+				}
+			},
+			{
+				"RU": {
+					"word": "мода",
+					"topics": ["одежда и обувь"]
+				}
+			}
+		],
+		[{
+				"EN": {
+					"word": "formal",
+					"topics": ["dress"],
+					"level": "elementary(A2)"
+				}
+			},
+			{
+				"RU": {
+					"word": "официальный",
+					"topics": ["одежда и обувь"]
+				}
+			}
+		],
+		[{
+				"EN": {
+					"word": "jewellery",
+					"topics": ["dress"],
+					"level": "elementary(A2)"
+				}
+			},
+			{
+				"RU": {
+					"word": "украшения",
+					"topics": ["одежда и обувь"]
+				}
+			}
+		],		[{
+				"EN": {
+					"word": "purse",
+					"topics": ["dress"],
+					"level": "elementary(A2)"
+				}
+			},
+			{
+				"RU": {
+					"word": "сумочка",
+					"topics": ["одежда и обувь"]
+				}
+			}
+		],		[{
+				"EN": {
+					"word": "remove",
+					"topics": ["dress"],
+					"level": "elementary(A2)"
+				}
+			},
+			{
+				"RU": {
+					"word": "снимать",
+					"topics": ["одежда и обувь"]
+				}
+			}
+		],		[{
+				"EN": {
+					"word": "size",
```

Comment:
please merge with maths and meausure or science

Reply:
no, size for clothes might differ in other languages i.e. in Czech: velikost vs rozměr